### PR TITLE
 SC streaming eval cli integration

### PR DESCRIPTION
**Pull request by ibanesh**
- Some changes for better integration of streaming evaluate cli in seamless_communication repo.
- Also making some changes so that the help menu of simuleval cli is not blocked by th multiple `parse_known_args `call

### CODE REVIEW

File: simuleval/options.py [hunk 1]
```diff
@@ -147,17 +147,23 @@ def import_user_module(module_path):
...
+        )
 
     parser.add_argument(
         "--user-dir",
         default=None,
         help="path to a python module containing custom agents",
     )
-    args, _ = parser.parse_known_args()
-    if args.user_dir is not None:
-        import_user_module(args.user_dir)
```
Comment (annasun28) [inline simuleval/options.py:160]: hm, why remove this?

Comment (annasun28) [reply simuleval/options.py:160]: I think this is what's breaking the unit tests

**Review by annasun28 — APPROVED** [review]
Lgtm after fixing unit tests

File: simuleval/options.py (hunk 1 above)
Comment (ibanesh) [reply simuleval/options.py:160]: ah! Thanks Anna. Was removed by mistake.

**Pull request merged**